src/gui/previewselect.cpp

```diff
@@ -39,9 +39,13 @@
 #include "base/utils/misc.h"
 #include "previewlistdelegate.h"
 
-PreviewSelect::PreviewSelect(QWidget* parent, BitTorrent::TorrentHandle *const torrent)
+#define SETTINGS_KEY(name) "PreviewSelect/" name
```

Comment:
IMO, "PreviewSelectDialog" is more clear.

Reply:
I would also rename the class itself and containing files.

Reply:
done